test/mode/modeNormal.test.ts

```diff
@@ -268,6 +268,14 @@ suite("Mode Normal", () => {
     });
 
     newTest({
+      title: "Can handle 'ci[' spanning multiple lines",
```

Comment:
I agree, the underlying functionality is coverd and no idea to test it again. I'll suggest adding tests for corner cases then we can make sure that we don't break this feature if we want to modify the architecture somehow one day.

For example, per [Vim doc](http://vimdoc.sourceforge.net/htmldoc/motion.html#i[) for `a[`, `When used in Visual mode it is made characterwise.` while this case is linewise. Another case is `i"`, we need take escaped `"` into consideration.

Not sure did I cover this kind of corner cases perfectly when I implement `aw/iw`, but we can together push it to the limit, what do you think? I'm not saying your test cases really miss some corner cases(I'm just not sure).